extensions/commands/cci/cmd_export_all_versions.py

```diff
@@ -0,0 +1,130 @@
+import os
+import json
+import textwrap
+import yaml
+
+from conan.api.output import ConanOutput, cli_out_write
+from conan.cli.command import conan_command, OnceArgument
+from conan.errors import ConanException
+
+
+def output_json(result):
+    cli_out_write(json.dumps({
+        "exported": [repr(r) for r in result['exported']],
+        "failures": result['failures']
+    }))
+
+
+def output_markdown(result):
+    exported = result['exported']
+    failures = result['failures']
+    print(textwrap.dedent(f"""
+    ### Conan Export Results
+
+    Successfully exported {sum([len(versions) for versions in exported.values()])} versions from {len(exported.keys())} recipes while encountering {len(failures)} recipes that could not be exported; these are
+
+
+    <table>
+    <th>
+    <td> Recipe </td> <td> Reason </td>
+    </th>"""))
+
+    for key, value in failures.items():
+        print(textwrap.dedent(f"""
+            <tr>
+            <td> {key} </td>
+            <td>
+
+            ```txt
+            """))
+        print(f"{value}")
+        print(textwrap.dedent(f"""
+            ```
+
+            </td>
+            </tr>
+            """))
+
+    print("</table>")
+
+
+@conan_command(group="Conan Center Index", formatters={"text": lambda result: None, "json": output_json, "md": output_markdown})
+def export_all_versions(conan_api, parser, *args):
+    """
+    Export all version of either a single recipe, a list of recipes, or a global recipes folder
+    """
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('-n', '--name', action=OnceArgument, help="Name of the recipe to export")
+    group.add_argument('-l', '--list', action=OnceArgument, help="YAML file with list of recipes to export")
+    group.add_argument('-p', '--path', action=OnceArgument, help="Path of the global recipe folder")
```

Comment:
```suggestion
    group.add_argument('-p', '--path', action=OnceArgument, help="Path of the global recipe folder. This will export all recipes")
```